test/ProblemDetailsResponseFactoryTest.php

```diff
@@ -432,4 +434,41 @@ function () {
 
         $this->assertSame($this->response->reveal(), $response);
     }
+
+    public function testRenderWithMalformedUtf8Sequences(): void
+    {
+        $e = $this->prophesize(RuntimeException::class)->willImplement(ProblemDetailsExceptionInterface::class);
+        $e->getStatus()->willReturn(400);
+        $e->getDetail()->willReturn('Exception details');
+        $e->getTitle()->willReturn('Invalid client request');
+        $e->getType()->willReturn('https://example.com/api/doc/invalid-client-request');
+        $e->getAdditionalData()->willReturn([
+            'malformed-utf8' => self::UTF_8_INVALID_2_OCTET_SEQUENCE,
+            ]);
```

Comment:
This line should have one level less indentation (should line up with the line where the array was opened).

Reply:
No clue how that slipped phpcs and my eyes. Fixed.